students/jjakub/session02/series.py

```diff
@@ -0,0 +1,59 @@
+# Fibonacci Function
+def fibonacci(n):
+    """Returns the nth value in the fibonacci series (starting with zero)"""
+    if n<2:
+        return n
+    else:
+        return fibonacci(n-1)+fibonacci(n-2)
+
+fib_result = fibonacci(8) # Input
+print(fib_result)
+
+# Lucas Function
+def lucas(n):
+    """Returns the nth value in the lucas series (starting with zero)"""
+    if n==0:
+        return 2
+    elif n==1:
+        return 1
+    else:
+        return lucas(n-1)+lucas(n-2)
+
+luc_result = lucas(8) # Input
+print(luc_result)
+
+# Sum Series
+def sum_series(n, val01=0, val02=1):
```

Comment:
Try to use more descriptive variable names. It is simple right now, but once the logic gets more complex, you want to avoid confusion and possibly wrong assumptions because of variable names.